### PR TITLE
Add docker compose with privadoid registry images

### PR DESCRIPTION
**Pull request by martinsaporiti**

### CODE REVIEW

File: infrastructure/local/docker-compose-full.yml [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+  postgres:
+    image: postgres:14-alpine
+    hostname: postgres
+    environment:
+      - PGPORT=5432
+      - PGUSER=postgres
+      - POSTGRES_HOST_AUTH_METHOD=trust
+      - POSTGRES_USER=postgres
+    ports:
+      - "5432:5432"
```
Comment (OBrezhniev) [inline infrastructure/local/docker-compose-full.yml:11]: please remove port forwarding, this is a common issue with the issuer node setup that users have another postgres already running. Instead we can "expose" ports to other containers in the project: https://docs.docker.com/reference/compose-file/services/#expose

File: infrastructure/local/docker-compose-full.yml [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+    healthcheck:
+      test: [ "CMD-SHELL", "pg_isready" ]
+      interval: 10s
+      timeout: 5s
+      retries: 5
+
+  redis:
+    image: redis:6-alpine
+    ports:
+      - "6379:6379"
```
Comment (OBrezhniev) [inline infrastructure/local/docker-compose-full.yml:24]: same here

File: infrastructure/local/docker-compose-full.yml [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+      timeout: 5s
+      retries: 5
+    depends_on:
+      - api
+
+  pending_publisher:
+    image: privadoid/issuernode
+    pull_policy: always
+    ports:
+      - "3005:3005"
```
Comment (OBrezhniev) [inline infrastructure/local/docker-compose-full.yml:88]: probably port doesn't need to be exposed

File: infrastructure/local/docker-compose-full.yml [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+    command: sh -c "./migrate && ./platform"
+    depends_on:
+      - postgres
+      - redis
+
+  notifications:
+    image: privadoid/issuernode
+    pull_policy: always
+    ports:
+      - "3004:3004"
```
Comment (OBrezhniev) [inline infrastructure/local/docker-compose-full.yml:70]: Do we need to expose them?

Comment (martinsaporiti) [reply infrastructure/local/docker-compose-full.yml:70]: we don't

**Pull request merged**